Solutions/ZoomReports/Data Connectors/ZoomSentinelConnector/__init__.py

```diff
@@ -148,35 +271,73 @@ def post_data(self, body, chunk_count):
             logging.info("Chunk was processed({} events)".format(chunk_count))
             self.success_processed = self.success_processed + chunk_count
         else:
-            logging.error("Error during sending events to Microsoft Sentinel. Response code:{}".format(response.status_code))
+            logging.error("Error during sending events to Microsoft Sentinel. Response code:{}".format(
+                response.status_code))
             self.fail_processed = self.fail_processed + chunk_count
 
-def results_array_join(result_element,api_req_id,api_req_name):
+
+def results_array_join(result_element, api_req_id, api_req_name):
+    """This method is used to join all the results from zoom api
+
+    Args:
+        result_element (_type_): This will have result element
+        api_req_id (_type_): zoom api req id
+        api_req_name (_type_): zoom api req name
+    """    
     for element in result_element[api_req_id]:
         element['event_type'] = api_req_id
         element['event_name'] = api_req_name
         results_array.append(element)
 
-def get_main_info():
+def check_if_functiontime_is_over(start_time, interval_minutes, max_script_exec_time_minutes):
+    """Returns True if function's execution time is less than interval between function executions and
+    less than max azure func lifetime. In other case returns False."""
+    
+    logging.info("started Max function time check")
+    max_minutes = min(interval_minutes, max_script_exec_time_minutes)
```

Comment:
Update it to min_minutes

Reply:
Done the changes, working on local testing with mock end point